.travis/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# THIS FILE IS MANAGED BY MODULESYNC
+
+rm -f Gemfile.lock
+if [ "${PUPPET_VERSION}" = '~> 4.0' ]; then
```

Comment:
since we're dropping version 4, no need for the conditional logic

Reply:
I will remove it in the next modulesync version (this is currently based on a template from https://github.com/voxpupuli/modulesync_config/tree/master/moduleroot )